keyboards/skme/zeno/config.h

```diff
@@ -17,13 +17,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
+/* key matrix size */
```

Comment:
You shouldnt be putting this and many others back, they have been removed on purpose.

Reply:
Pragma removed. I tried moving matrix configs to info but i could not compile firmware once i had done it.

Reply:
No, you are adding back matix size, debounce, polling interval, mcu, bootloader. All of the above are wrong additions and need to be reverted.

You should really review your proposed changes under the "Files Changed" tab.

Reply:
This is fixed